src/components/MySquads.js

```diff
@@ -183,7 +184,7 @@ const styles = StyleSheet.create({
   },
   icon: {
     padding: 3,
-    aspectRatio: "1/1",
+    aspectRatio: 1,
```

Comment:
For some reason, Android hated the fact that this was a string. Changing it to an integer seemed to fix it.

Reply:
Huh, that is interesting!

Reply:
<img width="361" alt="Aspect Ratio Error" src="https://user-images.githubusercontent.com/102986835/197352318-6df0d7c9-0e21-4451-9d89-a1900e541c02.png">